### ISSUE TITLE
socialData emulation

### ISSUE THREAD

**Issue by kirillbunin** [gameeapp/gamee-emulator#1]
Emulator does not have any option or mock data for 'socialData' emulation.

**Comment by EntityB** [comment]
Yes, emulator currently doesn't support socialData method at all. 

But we are currently working on it and it should be a part of the next emulator release ;) I marked the issue as an enhancement.

**Comment by kirillbunin** [comment]
How is this going?
I've got some feedback from your team, that there is in fact [working example of socialData](http://emulator-devel.gameeapp.com/emulator.html?projectUrl=https://d3d88ilde9yc1j.cloudfront.net/games/game-271/data/index.html) in emulator, but whenever i try to use `gamee.requestSocial` i get 
```
Uncaught ReferenceError: avatar is not defined
    at postMessageRequest (gamee-tools.js:2)
    at gamee-tools.js:2
```

**Comment by EntityB** [comment]
Hey yeah, but it is 
http://emulator-devel.gameeapp.com
and this link is not mentioned to be used for testing games, but if it helps...
